From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: v8-ci-autoroll-builder
 <v8-ci-autoroll-builder@chops-service-accounts.iam.gserviceaccount.com>
Date: Tue, 13 Dec 2022 20:01:56 +0000
Subject: Update V8 to version 11.0.218.

Summary of changes available at:
https://chromium.googlesource.com/v8/v8/+log/e31afd1a..54804f29

Please follow these instructions for assigning/CC'ing issues:
https://v8.dev/docs/triage-issues

Please close rolling in case of a roll revert:
https://v8-roll.appspot.com/
This only works with a Google account.

CQ_INCLUDE_TRYBOTS=luci.chromium.try:linux-blink-rel
CQ_INCLUDE_TRYBOTS=luci.chromium.try:linux_optional_gpu_tests_rel
CQ_INCLUDE_TRYBOTS=luci.chromium.try:mac_optional_gpu_tests_rel
CQ_INCLUDE_TRYBOTS=luci.chromium.try:win_optional_gpu_tests_rel
CQ_INCLUDE_TRYBOTS=luci.chromium.try:android_optional_gpu_tests_rel

R=hablich@chromium.org,vahl@chromium.org,v8-waterfall-sheriff@grotations.appspotmail.com

Change-Id: I93a95971034a9688752445fe5fe00426bb2b6288
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4098391
Reviewed-by: Adam Klein <adamk@chromium.org>
Commit-Queue: Adam Klein <adamk@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1082670}

diff --git a/DEPS b/DEPS
index b075b72d48ea6df9ee9b99a7760b78fc00d2d253..3b6f43329474aad615e5bc35e19da5b4e1099f8f 100644
--- a/DEPS
+++ b/DEPS
@@ -299,7 +299,7 @@ vars = {
   # Three lines of non-changing comments so that
   # the commit queue can handle CLs rolling V8
   # and whatever else without interference from each other.
-  'v8_revision': '9f0f79a53f410e52da4c5827fa49a5f038776379',
+  'v8_revision': '54804f2912112706ea3c13a73a460a14ba9bbb89',
   # Three lines of non-changing comments so that
   # the commit queue can handle CLs rolling ANGLE
   # and whatever else without interference from each other.
